x-pack/plugins/security_solution/public/common/lib/kibana/kibana_react.mock.ts

```diff
@@ -113,6 +114,12 @@ export const createStartServicesMock = (): StartServices => {
     },
     security,
     storage,
+    ml: {
```

Comment:
Not critical, but: it would be nice if ml provided a mock for this contract!

Reply:
Thanks Ryland! Updated here https://github.com/elastic/kibana/pull/78209/commits/1c390c169993ec2f6d687a52752a46db98cb8f27